api/services/References.js

```diff
@@ -8,7 +8,8 @@ exports.approve = function (ref, approve) {
       or: [
         {type: 'casual'},
         {type: 'shiny'},
-        {type: 'event'}
+        {type: 'event'},
+        {type: 'bank'}
```

Comment:
We could probably keep it bank in the backend full-stop, because an actual change seems unnecessary. There are other things that are still called their original names. It's obvious what they actually are.